animareflection/ui
# Ark v1 migration

## Description

##### Task link: https://trello.com/c/NKiThzt9/296-ark-v1-migration

- Migrated internal Ark-based component APIs to Ark v1
  - Updated tests accordingly
  - Note that the only user-facing API change is to `Carousel` (`slides` prop → `items`)

## Test Steps

- Verify components render as expected in Storybook
- Verify usage in the example apps renders as expected
- Carefully check to see if I missed any API mappings (as in make sure our API export names match Ark's API, e.g. `CarouselNextSlideTrigger` -> `CarouselNextTrigger`)
